### PR TITLE
[LMENGA-1138] Bump node and other dependencies

### PR DESCRIPTION
**Pull request by ninablanson-p44**
# WHAT & WHY
- Bump the node version to 18.2.0
- Clean up dependencies

# PROOF
## Docker build succeeds
Screen Shot 2022-05-25 at 4 50 04 PM

### CODE REVIEW

File: node-react/package.json [hunk 1]
```diff
@@ -3,16 +3,11 @@
...
   "description": "A NodeJS and React boilerplate for frontend interviews",
   "private": true,
-  "scripts": {
-    "bootstrap": "lerna bootstrap"
-  },
   "workspaces": [
     "packages/*"
   ],
-  "devDependencies": {
-    "lerna": "^4.0.0"
```
Comment (ninablanson-p44) [inline node-react/package.json:13]: It doesn't look like we're using Lerna any longer, so I deleted it.

File: node-react/packages/backend/package.json [hunk 2]
```diff
@@ -2,27 +2,23 @@
...
-    "express": "^4.17.3"
+    "express": "^4.18.1"
   },
   "devDependencies": {
-    "@babel/core": "^7.17.5",
-    "@babel/eslint-parser": "^7.17.0",
-    "@babel/node": "^7.16.8",
-    "@babel/plugin-proposal-class-properties": "^7.16.7",
-    "@babel/plugin-proposal-object-rest-spread": "^7.17.3",
-    "@babel/preset-env": "^7.16.11",
```
Comment (ninablanson-p44) [inline node-react/packages/backend/package.json:18]: Not 100% sure why we needed babel here, everything appears to work without it. @branclar if you can think of anything LMK.

File: node-react/packages/backend/prettier.config.cjs [hunk 3]
```diff
@@ -1,7 +1,6 @@
 module.exports = {
   arrowParens: 'avoid',
   bracketSpacing: true,
-  jsxBracketSameLine: false,
```
Comment (ninablanson-p44) [inline node-react/packages/backend/prettier.config.cjs:4]: Deprecated, also not relevant to backend.

File: node-react/packages/frontend/package.json [hunk 4]
```diff
@@ -3,12 +3,8 @@
...
   "private": true,
   "dependencies": {
-    "@testing-library/jest-dom": "^5.16.2",
-    "@testing-library/react": "^12.1.3",
-    "@testing-library/user-event": "^13.5.0",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-scripts": "5.0.0",
+    "react": "^18.1.0",
+    "react-dom": "^18.1.0",
```
Comment (ninablanson-p44) [inline node-react/packages/frontend/package.json:7]: 🎵 _Fancy, oooo_ 🎵

File: node-react/packages/frontend/package.json [hunk 5]
```diff
@@ -3,12 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@testing-library/jest-dom": "^5.16.2",
-    "@testing-library/react": "^12.1.3",
-    "@testing-library/user-event": "^13.5.0",
```
Comment (ninablanson-p44) [inline node-react/packages/frontend/package.json:8]: Moved these to be testing dependencies.

File: node-react/packages/frontend/prettier.config.js [hunk 6]
```diff
@@ -1,7 +1,7 @@
 module.exports = {
   arrowParens: 'avoid',
+  bracketSameLine: false,
```
Comment (ninablanson-p44) [inline node-react/packages/frontend/prettier.config.js:3]: New name for the `jsxBracketSameLine` config.

File: node-react/packages/frontend/yarn.lock [hunk 7]
```diff
@@ -1,8809 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (ninablanson-p44) [inline node-react/packages/frontend/yarn.lock:1]: Having a separate yarn.lock file is unnecessary for workspaces.

**Review by Ashley-p44 — APPROVED** [review]

**Pull request merged**